foxsec-slack-bot/slackbot.go

```diff
@@ -62,7 +75,20 @@ func InitConfig() {
 		log.Fatalf("Could not create kms client. Err: %s", err)
 	}
 
-	log.Infof("Decrypting with key: %s", KEYNAME)
+	accessKeyId, err := kms.DecryptEnvVar(KEYNAME, "AWS_ACCESS_KEY_ID")
+	if err != nil {
+		log.Fatalf("Could not decrypt aws access key. Err: %s", err)
```

Comment:
Is it possible the encrypted secret could end up in the logs like this?

Reply:
I don't think so, from looking at the code it just includes the error response from the API in the returned `error`.